src/python/grpcio_tests/tests/protoc_plugin/_python_plugin_test.py

```diff
@@ -237,6 +237,7 @@ def testUpDown(self):
         self.assertIsNotNone(service.servicer_methods)
         self.assertIsNotNone(service.server)
         self.assertIsNotNone(service.stub)
+        service.server.stop(None)
```

Comment:
I think the changes to this file and `_split_definitions_test` are worth breaking out into a separate pull request.

My apologies for not calling it out earlier in review.

Reply:
I think there is important context to this change in this PR.  I've broken it into a separate commit for some separation though.